JSDemos/Demos/DataGrid/ToolbarCustomization/description.md

```diff
@@ -1,12 +1,12 @@
-The DataGrid uses an integrated DevExtreme [Toolbar](/Demos/WidgetsGallery/Demo/Toolbar/Overview/) component to display predefined and custom commands. To add or remove toolbar items, implement the [onToolbarPreparing](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onToolbarPreparing) function. Use its parameter's **toolbarOptions** field to access the [Toolbar's configuration](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/) object. This object includes the command collection: the **toolbarOptions**.[items](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/) array.
+DataGrid's toolbar allows you to customize predefined controls and add DevExtreme components and custom elements. To do this, use the [toolbar](Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/).[items[]](Documentation/21_2/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/) array. Declare the array elements in the order the controls should have in the UI.
 
-This demo illustrates how to add the following items to the toolbar:
+### Predefined Controls        
+An added DataGrid feature makes the corresponding predefined control available. If you need to customize it, add an object to the **items[]** array. This object must contain the control's [name](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/toolbar/items/#name) and [properties](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/toolbar/items/) that you want to customize. If a control does not need customization, simply include its **name** in the **items[]** array. 
```

Comment:
> An added DataGrid feature makes the corresponding predefined control available.

"Added" where? "Available" where? Why rewrite the original sentence?

> If you need to customize it

Customize the feature or the control?